README.md

```diff
@@ -1,1134 +1,233 @@
+project. This project is `swagger-tester`.  To test your changes as you're developing the plugin, cd into this 
 directory, and run sbt. This project uses an sbt `ProjectRef` to the sbt plugin, which means you don't need to 
 `publishLocal` the plugin after each change. Just run `reload` in the sbt console, and it will pick up your changes.
 
-The Api-First-Hand plugin provides a couple of commands useful for development: 
-
-* `apiFirstPrintDenotations` - outputs a common names of different parts of the AST as they are intended to be used in generated Scala code
-* `apiFirstPrintRawAstTypes` - outputs all type definitions as they read from the specification before type optimisations
-* `apiFirstPrintRawAstParameters` - outputs all parameters definitions before type optimisations
-* `apiFirstPrintFlatAstTypes` - outputs type definitions after type optimisations
-* `apiFirstPrintFlatAstParameters` - outputs parameter definitions after type optimisations
+API-First-Hand provides a few commands useful for development: 
 
+* `apiFirstPrintDenotations`: outputs common names of different parts of the AST as they are intended for use in generated Scala code
+* `apiFirstPrintRawAstTypes`: outputs all type definitions as they read from the specification before type optimizations
+* `apiFirstPrintRawAstParameters`: outputs all parameters definitions before type optimizations
+* `apiFirstPrintFlatAstTypes`: outputs type definitions after type optimizations
+* `apiFirstPrintFlatAstParameters`: outputs parameter definitions after type optimizations
 
 ## Plugin Testing
 
-We're using the sbt scripted framework for testing. You can find the tests in `plugin/src/sbt-test`, and run them 
+We're using the sbt scripted framework for testing. You can find the tests in `plugin/src/sbt-test` and run them 
 by running `scripted` in the sbt console.
 
+## Code Quality
 
-## Code quality
-
-There are some quality checks embedded into the build script:
+Some quality checks are embedded into the build script:
 * the source code is (re)formatted using [scalariform](https://github.com/scala-ide/scalariform) each time it is compiled (currently deactivated).
-* [`scalastyle`](http://www.scalastyle.org) sbt command shall be used to perform code style checks before putting changes into the repository.
-* [`lint:compile`](https://github.com/HairyFotr/linter) sbt command shall be used to perform static code analysis before putting changes into the repository.
-* code coverage for api and compiler modules can be executed by issuing `sbt clean coverage test` command for these 
-projects. Coverage statistics can be generated using `coverageReport` sbt command. 
+* [`scalastyle`](http://www.scalastyle.org) sbt command performs code style checks before putting changes into the repository
+* [`lint:compile`](https://github.com/HairyFotr/linter) sbt command performs static code analysis before putting changes into the repository
```

Comment:
sounds like it happens automatically before commit or push. It's not as far as I know, previous versions seems more correct. Maybe just remove "before putting changes into the repository"